tools/check_languages.py

```
@@ -0,0 +1,53 @@
+def open_lang(name):
+    parser = configparser.ConfigParser()
+    parser.optionxform = lambda str:str
+    parser.read(LANG_DIR + name, "utf-8")
+    return parser
+
+DEF_NAME = "English.ini"
+DEF_LANG = open_lang(DEF_NAME)
+
+for file in os.listdir(LANG_DIR):
+    filename = os.fsdecode(file)
+    if filename == DEF_NAME: continue
```

Ruff (0.15.17)

[error] Multiple statements on one line (colon)

(E701)